runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTileFooter/ProcessTileFooter.types.ts

```diff
@@ -3,3 +3,5 @@ import { IProcess } from 'runbotics-common';
 export interface ProcessTileFooterProps {
     process: IProcess;
 }
+
+export type Color = 'primary' | 'secondary' | 'error' | 'warning' | 'info' | 'success' | 'grey';
```

Comment:
imo grey shouldn't be part of so general union

Reply:
Why? All colors can be used in footer. And additionally gray is necessary in mock.